tools/flow9/backends/link.flow

```diff
@@ -1,16 +1,22 @@
 import tools/flow9/backends/rename_ids;
 import tools/flow9/backends/bprogram_lower;
 import tools/flow9/driver;
+import tools/flow9/backends/ref_counting;
 
 export {
 	linkBExpProgram(cache : FlowCache, flowpath : string, shorten_ids : bool, spec : BackendSpec) -> BExpProgram;
 	linkBProgram(cache : FlowCache, flowpath : string, shorten_ids : bool, spec : BackendSpec) -> BProgram;
 	programBModules(cache : FlowCache, file : string, acc : List<BModule>, visited : Set<string>) -> Pair<List<BModule>, Set<string>>;
+	memoryManagement = isUrlParameterTrue("mem");
```

Comment:
I have used "Tracing" for a place to keep flags before.

Reply:
the flag is mainly used to generate C ++ code in a new format, not to print debug information. the flag will be removed in the future. (it is needed to compare versions of c ++ files.)